### PR TITLE
Adjust styles for mobile

### PR DESCRIPTION
**Pull request by OlegMoshkovich**
- About: Adjust the highlighted project for mobile to fit the dialog
- Snackbar: defined width for mobile
- Logo: Add a link back to BLDRS
- Properties Drawer: Simplified styles - reduced the height of the drawer
- Cutplane Icon: Only show on mobile

https://olegmoshkovich.github.io/Share/share/v/p/index.ifc#c:-111.37,14.94,90.63,-43.48,15.73,-4.34

image

image

image

image

### CODE REVIEW

**Comment by pablo-mayrgundter** [comment]
Nice!  Only issue is that the click-on-logo takes you to the desktop camera, not the mobile.  To get desktop users back to desktop camera and mobile back to mobile camera I think you should call Share.jsx#navToDefault

**Comment by OlegMoshkovich** [comment]
ok got it.

**Comment by pablo-mayrgundter** [comment]
Hm, I see a couple issues.  On mobile, I'm getting a broken About button:

image

I don't see this on web, but related, the SVG you're using has hard-coded background white disc and fill colors.  Please remove these and test with varying themes.

File: src/assets/2D_Icons/Wave_person.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg width="50" height="50" viewBox="0 0 50 50" fill="none" xmlns="http://www.w3.org/2000/svg">
+<circle cx="25" cy="25.0001" r="25" fill="white"/>
```
Comment (pablo-mayrgundter) [inline src/assets/2D_Icons/Wave_person.svg:2]: remove this line

File: src/assets/2D_Icons/Wave_person.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+<path d="M25 17C26.6569 17 28 15.6569 28 14C28 12.3431 26.6569 11 25 11C23.3431 11 22 12.3431 22 14C22 15.6569 23.3431 17 25 17Z" fill="black"/>
```
Comment (pablo-mayrgundter) [inline src/assets/2D_Icons/Wave_person.svg:3]: remove fill colors from these, they should be filled by theme

**Review by pablo-mayrgundter — CHANGES_REQUESTED** [review]

File: src/assets/2D_Icons/Wave_person.svg (hunk 2 above)
Comment (OlegMoshkovich) [reply src/assets/2D_Icons/Wave_person.svg]: done

**Comment by OlegMoshkovich** [comment]
image
 @pablo-mayrgundter PTAL

**Review by pablo-mayrgundter — COMMENTED** [review]
Hello icon is better but still needs circle removed and also looks oval for me.. probably needs width and height set as with others.

**Pull request merged**